packages/expo/android/src/main/java/expo/modules/ExpoModulesPackage.kt

```diff
@@ -27,6 +28,10 @@ class ExpoModulesPackage : ReactPackage {
         emptyList()
       }
     }
+
+    val packagePrioritySortedList: List<Package> by lazy {
+      packageList.sortedByDescending { ModulePriorities.get(it::class.qualifiedName) }
+    }
```

Comment:
Do we need another method here? 🤔 
Can't `packageList` be always sorted using these `priorities`?
cc @tsapeta

Reply:
I don't have any opinion on this, so up to Kudo 😉 My only concern would be that someone may want to use its own package list (not the generated one) and then we have no control on the order, but this is very rare use case and I guess we don't need to care about that 😄 
EDIT: It's also probably easier to find out that they're being sorted that way from the native code rather than JS in autolinking, but I'm not sure if that matters that much.

Reply:
it would be clear in source code if there is just one single `packageList` and be sorted. i would merge `packagePrioritySortedList` into `packageList`.

for the ordering, i think our design just internally make sure our modules' in sorted order. other modules or 3rd party modules are just in random.